### DISCUSSION TITLE
Missing slider ticks should impact S grading

### DISCUSSION THREAD

**Discussion by PuddlePumpkin** [ppy/osu#26676] [Ideas] [answered]
Regarding osu! ruleset grading change #26630 originally discussed in #4184,
While playtesting this pull request I noticed that it feels very strange that a missed slider tick, which in terms of gameplay, is weighted as a bad enough mistake to cause a combo break, does **not** impact S grade qualification. I believe making slider tick misses invalidate S grading would make this change feel a lot more consistent.

**Proposed further changes:**
Slider tick combobreaks should result in A grade maximum.

**Comment by peppy** [comment, marked as answer]
Thanks for your feedback. We are copying what stable had and what people are used to.

**Reply by PuddlePumpkin** [reply]
I feel like this change could be a lot less controversial than allowing S on miss while also somewhat improving stables behavior. Most people (i think, speaking from playing osu! for 10 years) mostly just care about their fc's being marked with S. when we don't FC having an S or not doesn't really matter.

**Reply by peppy** [reply]
And for the record, you're also proposing slider ticks are required but not slider ends right?

If so that's a double standard and also becomes redundant in recent maps which set slider ticks to so rare you never see one.

**Reply by PuddlePumpkin** [reply]
Yes, since slider ends don't break combo it still feels like an FC / S when you complete the map.
Also, since slider ticks and ends have different behaviors with combo break, they already feel completely separate.
